spec/models/experience_point_award_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe ExperiencePointAward, type: :model do
+  let(:user){ User.create username: "mcfiredrill", time_zone: "Tokyo", role: "admin owner" }
+  describe "leveling up" do
+    xit "gains multiple levels at once"
```

Comment:
maybe we can test out the multiple levels at once here?

Reply:
Wil merge after I write this test